.drone.yml

```diff
@@ -7,13 +7,43 @@ pipeline:
     image: golang:1.9
     commands: sh .drone.sh
 
-  snapshot:
+  publish_latest:
     image: plugins/docker
     repo: drone/cli
     secrets: [docker_username, docker_password]
+    auto_tag: true
     when:
-      branch: master
-      event: push
+      event: [push, tag]
+
+  publish_alpine:
+    image: plugins/docker
+    repo: drone/cli
+    secrets: [docker_username, docker_password]
+    auto_tag: true
+    auto_tag_suffix: alpine
+    dockerfile: Dockerfile.alpine
+    when:
+      event: [push, tag]
+
+  publish_arm:
+    image: plugins/docker
+    repo: drone/cli
+    secrets: [docker_username, docker_password]
+    auto_tag: true
+    auto_tag_suffix: arm
+    dockerfile: Dockerfile.arm
+    when:
+      event: [push, tag]
+
+  publish_arm64:
+    image: plugins/docker
+    repo: drone/cli
+    secrets: [docker_username, docker_password]
+    auto_tag: true
+    auto_tag_suffix: arm64
```

Comment:
sorry, forgot to mention we are using `linux-arm64` convention so that if / when we support windows containers we won't hit a naming conflict